src/main/java/ganymedes01/etfuturum/client/DynamicSoundsResourcePack.java

```diff
@@ -88,43 +82,6 @@ private void addSoundsToCategory(String cat, String... sounds) {
 			}
 		}
 
-		private void addSoundEventsToCategory(String cat, String... sounds) {
-			addSoundsToCategoryWithSettings(cat, 1.0F, 1.0F, 1, false, true, sounds);
-		}
-
-		private void addSoundsToCategoryWithSettings(String cat, float volume, float pitch, String... sounds) {
-			addSoundsToCategoryWithSettings(cat, volume, pitch, 1, false, false, sounds);
-		}
-
-		private void addSoundsToCategoryWithSettings(String cat, float volume, float pitch, int weight, String... sounds) {
-			addSoundsToCategoryWithSettings(cat, volume, pitch, weight, false, false, sounds);
-		}
-
-		private void addSoundsToCategoryWithSettings(String cat, float volume, float pitch, int weight, boolean stream, boolean isEvent, String... sounds) {
-			JsonObject soundCat = JsonUtil.getOrCreateObject(rootObject, cat);
-			JsonArray soundList = JsonUtil.getOrCreateArray(soundCat, "sounds");
-			for (String sound : sounds) {
-				JsonObject soundObj = new JsonObject();
-				soundObj.add("name", new JsonPrimitive(sound));
-				if (volume != 1.0F) {
-					soundObj.add("volume", new JsonPrimitive(MathHelper.clamp_float(volume, 0, 1)));
-				}
-				if (pitch != 1.0F) {
-					soundObj.add("pitch", new JsonPrimitive(pitch));
-				}
-				if (weight > 1) {
-					soundObj.add("weight", new JsonPrimitive(weight));
-				}
-				if (stream) {
-					soundObj.add("stream", new JsonPrimitive(true));
-				}
-				if (isEvent) {
-					soundObj.add("type", new JsonPrimitive("event"));
-				}
-				soundList.add(soundObj);
-			}
-		}
-
```

Comment:
Unused ≠ Dead
Please refrain from removing utility code that may be used in the future.